package.json

```diff
@@ -24,7 +24,8 @@
     "min-js": "google-closure-compiler-js dist/radi.js > dist/radi.min.js",
     "min-es": "squash dist/radi.es.js -o dist/radi.es.min.js",
     "lint": "eslint ./src/**/*.js --fix",
-    "test": "jest --watch",
+    "test:watch": "jest --watch",
```

Comment:
I assume that combine `npm run build` and `npm run test` is good choice for CI. For example,  `"ci": "npm run test:ci && npm run build"` where  `"test:ci": "npm run test -- --ci"`

Please see [ci](https://facebook.github.io/jest/docs/en/cli.html#ci)